external/synth/synth/clis.py

```diff
@@ -0,0 +1,14 @@
+import argparse
+from . import core
+
+
+def read_schema():
```

Comment:
This does more than just "read" a schema.  Would something like `create_schema` or `copy_schema` be a better name?

Reply:
Renamed to "copy_schema"